stripe.go

```diff
@@ -779,6 +835,9 @@ const minNetworkRetriesDelay = 500 * time.Millisecond
 
 const uploadsURL = "https://uploads.stripe.com"
 
+// The number of requestMetric objects to buffer for client telemetry.
+const telemetryBufferSize = 16
```

Comment:
Can you move this up a couple lines? (The constant names are ordered alphabetically so let's try to keep them that way).

Reply:
Also, would you mind amending the comment to mention that additional objects will be dropped if the buffer is full?  It'll make it extra clear what the lib does when the buffer is full (as opposed to other behaviors like growing the buffer, or sending metrics synchronously, etc.)

(I know you mention that at the site where the channel is actually written to, but I think it'd be helpful to have here as well).